src/renderer/components/dock/install-chart/install-chart-model.injectable.tsx

```diff
@@ -0,0 +1,281 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable, lifecycleEnum } from "@ogre-tools/injectable";
+import installChartTabStoreInjectable from "./store.injectable";
+import { waitUntilDefined } from "../../../../common/utils";
+import type { CallForHelmChartValues } from "./chart-data/call-for-helm-chart-values.injectable";
+import callForHelmChartValuesInjectable from "./chart-data/call-for-helm-chart-values.injectable";
+import type { IChartInstallData, InstallChartTabStore } from "./store";
+import type { HelmChart } from "../../../../common/k8s-api/endpoints/helm-charts.api";
+import React from "react";
+import {
+  action,
+  computed,
+  observable,
+  runInAction,
+} from "mobx";
+import assert from "assert";
+import type { CallForCreateHelmRelease } from "../../+helm-releases/create-release/call-for-create-helm-release.injectable";
+import callForCreateHelmReleaseInjectable from "../../+helm-releases/create-release/call-for-create-helm-release.injectable";
+import type { HelmReleaseUpdateDetails } from "../../../../common/k8s-api/endpoints/helm-releases.api";
+import dockStoreInjectable from "../dock/store.injectable";
+import type { NavigateToHelmReleases } from "../../../../common/front-end-routing/routes/cluster/helm/releases/navigate-to-helm-releases.injectable";
+import navigateToHelmReleasesInjectable from "../../../../common/front-end-routing/routes/cluster/helm/releases/navigate-to-helm-releases.injectable";
+import type { SingleValue } from "react-select";
+import type { CallForHelmChartVersions } from "../../+helm-charts/details/versions/call-for-helm-chart-versions.injectable";
+import callForHelmChartVersionsInjectable from "../../+helm-charts/details/versions/call-for-helm-chart-versions.injectable";
+
+const installChartModelInjectable = getInjectable({
+  id: "install-chart-model",
+
+  instantiate: async (di, tabId: string) => {
+    const store = di.inject(installChartTabStoreInjectable);
+    const callForHelmChartValues = di.inject(callForHelmChartValuesInjectable);
```

Comment:
comment: what is the benefit of naming these sort of injectables with the `callFor` prefix? Wouldn't `getHelmChartValues` be descriptive enough?

Reply:
It's just matter of preference and what we're used to. 

We've been using `callForX` for stuff that needs to be get from somewhere outside of the system. E.g. API call, file system, local storage. `getX` doesn't really communicate enough because it could also mean that we are getting some static value from somewhere or e.g. some deeply nested value from the input.